plugin/pkg/scheduler/factory/plugins_test.go

```diff
@@ -39,3 +39,22 @@ func TestAlgorithmNameValidation(t *testing.T) {
 		}
 	}
 }
+
+func TestPriorityOverFlow(t *testing.T) {
```

Comment:
I thought about that but I guess that should be part of other test that calls the checkOverFlow function. I don't have strong opinion on this.